examples/apps/attributable-map/src/view.ts

```diff
@@ -0,0 +1,141 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IRuntimeAttributor } from "@fluid-experimental/attributor";
+import { IHitCounter, greenKey, redKey, ITinyliciousUser } from "./dataObject";
+
+export function renderHitCounter(
+	hitCounter: IHitCounter,
+	runtimeAttributor: IRuntimeAttributor | undefined,
+	div: HTMLDivElement,
+) {
+	// Create wrapper div and add styling
+	const wrapperDiv = document.createElement("div");
```

Comment:
all of this dom element insertion is pretty rough for readability, when the majority of it isn't relevant for the javascript. Could you put it in the static HTML instead and put classnames on the parts that the JS needs to interact with, querying for them?